km-sidekiq.gemspec

```diff
@@ -2,6 +2,8 @@
 require File.expand_path('../lib/km_sidekiq/version', __FILE__)
 
 Gem::Specification.new do |gem|
+  gem.required_ruby_version = '>= 3.0.0'
```

Comment:
Probably not needed or too restrictive

Reply:
Can we get away without it?

Reply:
For our own needs yes.

If someone uses it with an old version of ruby (likely < 1.9.2 based on https://apidock.com/ruby/URI/encode_www_form/class), they may run into a problem.